tests/functional/test_install.py

```diff
@@ -1884,3 +1884,9 @@ def test_install_skip_work_dir_pkg(script, data):
 
     assert 'Requirement already satisfied: simple' not in result.stdout
     assert 'Successfully installed simple' in result.stdout
+
+
+def test_install_logs_pip_version_in_debug(script):
+    result = script.pip('install', '-v', 'INITools==0.2')
```

Comment:
This test uses the network. I'd suggest to install a package from the fixture `shared_data`.